Failed SvgIconTypeMap import

eslint --fix

```
import {SvgIconTypeMap} from "@mui/material";
↓
import SvgIconTypeMap from "@mui/material/SvgIconTypeMap";
```

throw Error
Cannot find module '@mui/material/SvgIconTypeMap' or its corresponding type declarations.ts(2307)

--- a/lib/rules/mui-path-imports.js
+++ b/lib/rules/mui-path-imports.js
@@ -27,34 +27,47 @@ module.exports = {
   create(context) {
     return {
       ImportDeclaration(node) {
-        if (!["@mui/material", "@mui/icons-material"].includes(node.source.value)) return
+        const validationTargets = [
+          "@mui/material", "@mui/icons-material"
+        ]
+        const allowNamedImports = [
+          "Theme", "SvgIconTypeMap"
+        ]
+        if (!validationTargets.includes(node.source.value)) {
+          return
+        }
         if (
           node.source.value === '@mui/material' &&
-          node.specifiers.length === 1 &&
-          node.specifiers[0].local.name === "Theme"
-        ) return
+          node.specifiers.length > 0 &&
+          node.specifiers.every(specifier => {
+            return allowNamedImports.includes(specifier.local.name)
+          })
+        ) {
+          return
+        }
 
-        const isNotDefaultImport = node.specifiers.some(specifier => {
+        const hasError = node.specifiers.some(specifier => {
           if (specifier.type !== "ImportDefaultSpecifier") return true
           return false
         })
-        if (isNotDefaultImport) {
+        if (hasError) {
           context.report({
             node,
             message: "error: !mui-toplevel-import",
             fix(fixer) {
-              const a = node.specifiers.filter((specifier) => {
-                return specifier.local.name !== "Theme"
-              }).map(specifier => {
-                return `import ${specifier.local.name} from "${node.source.value}/${specifier.local.name}";`
+              const fixed = []
+              node.specifiers.filter((specifier) => {
+                return !allowNamedImports.includes(specifier.local.name)
+              }).forEach(specifier => {
+                fixed.push(`import ${specifier.local.name} from "${node.source.value}/${specifier.local.name}";`)
               })
               const b = node.specifiers.filter((specifier) => {
-                return specifier.local.name === "Theme"
-              }).map(specifier => {
-                return `import { ${specifier.local.name} } from "${node.source.value}";`
+                return allowNamedImports.includes(specifier.local.name)
               })
-              const c = [...a, ...b].join("\n")
-              return fixer.replaceText(node, c)
+              if (b.length > 0) {
+                fixed.push(`import { ${b.map(bb => bb.local.name).join(", ")} } from "${node.source.value}";`)
+              }
+              return fixer.replaceText(node, fixed.join("\n"))
             }
           })
         }

--- a/tests/lib/rules/mui-path-imports.js
+++ b/tests/lib/rules/mui-path-imports.js
@@ -30,7 +30,16 @@ ruleTester.run("rule", rule, {
       code: `import { Theme } from "@mui/material";`
     },
     {
-      code: `import { Box } from "other/package";`
+      code: `import { SvgIconTypeMap } from "@mui/material";`
+    },
+    {
+      code: `import { SvgIconTypeMap, Theme } from "@mui/material";`
+    },
+    {
+      code: `import { Theme, SvgIconTypeMap } from "@mui/material";`
+    },
+    {
+      code: `import { Other } from "other/package";`
     },
   ],
 
@@ -62,6 +71,24 @@ import Card from "@mui/material/Card";`
       errors: [{ message: "error: !mui-toplevel-import", type: "" }],
       output: `import Box from "@mui/material/Box";
 import { Theme } from "@mui/material";`,
+    },
+    {
+      code: `import { Box, Theme, SvgIconTypeMap } from "@mui/material";`,
+      errors: [{ message: "error: !mui-toplevel-import", type: "" }],
+      output: `import Box from "@mui/material/Box";
+import { Theme, SvgIconTypeMap } from "@mui/material";`
+    },
+    {
+      code: `import { Theme, Box, SvgIconTypeMap } from "@mui/material";`,
+      errors: [{ message: "error: !mui-toplevel-import", type: "" }],
+      output: `import Box from "@mui/material/Box";
+import { Theme, SvgIconTypeMap } from "@mui/material";`
+    },
+    {
+      code: `import { Theme, SvgIconTypeMap, Box } from "@mui/material";`,
+      errors: [{ message: "error: !mui-toplevel-import", type: "" }],
+      output: `import Box from "@mui/material/Box";
+import { Theme, SvgIconTypeMap } from "@mui/material";`
     },
   ],
 });

